.travis.yml

```diff
@@ -14,7 +18,5 @@ after_success:
   - coveralls
 notifications:
   email:
-    recipients:
-      - yukino0131@me.com
     on_success: always
     on_failure: always
```

Comment:
Had to drop this on my testing. Not sure if this is needed as this is the default on travis no?